src/sql/src/statements.rs

```diff
@@ -531,4 +535,61 @@ mod tests {
         )
         .is_err());
     }
+
+    #[test]
+    pub fn test_parse_column_default_constraint() {
+        let bool_value = sqlparser::ast::Value::Boolean(true);
+
+        let opts = vec![
+            ColumnOptionDef {
+                name: None,
+                option: ColumnOption::Default(Expr::Value(bool_value)),
+            },
+            ColumnOptionDef {
+                name: None,
+                option: ColumnOption::NotNull,
+            },
+        ];
+
+        let constraint =
+            parse_column_default_constraint("coll", &ConcreteDataType::Boolean(BooleanType), &opts)
+                .unwrap();
+
+        assert_matches!(
+            constraint,
+            Some(ColumnDefaultConstraint::Value(Value::Boolean(true)))
+        );
+    }
+
+    #[test]
+    pub fn test_sql_column_def_to_grpc_column_def() {
+        // test basic
+        let column_def = ColumnDef {
+            name: "col".into(),
+            data_type: DataType::Double,
+            collation: None,
+            options: vec![],
+        };
+
+        let grpc_column_def = sql_column_def_to_grpc_column_def(column_def).unwrap();
+
+        assert_eq!("col", grpc_column_def.name);
+        assert!(grpc_column_def.is_nullable); // nullable when options are empty
+        assert_eq!(10, grpc_column_def.datatype);
```

Comment:
Better to use `ColumnDataType::Float64 as i64`

Reply:
thanks~ updated @waynexia 

And I removed hardcode number in src/datanode/src/server/grpc/ddl.rs too, do you think I should create another PR  for ddl?

Reply:
Well done! It's ok to include this change 🥰